build/ova-provider-server/Containerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21.11-9 AS builder
+WORKDIR /app
+COPY --chown=1001:0 ./ ./
+ENV GOFLAGS "-mod=vendor -tags=strictfipsruntime"
+ENV GOEXPERIMENT strictfipsruntime
+
+RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o ova-provider-server github.com/konveyor/forklift-controller/cmd/ova-provider-server
+
+FROM registry.access.redhat.com/ubi9-minimal:9.4-1227.1725849298
+# Required to be able to get files from within the pod
+RUN microdnf -y install tar && microdnf clean all
+
+COPY --from=builder /app/ova-provider-server /usr/local/bin/ova-provider-server
+ENTRYPOINT ["/usr/local/bin/ova-provider-server"]
+
+LABEL \
+        com.redhat.component="forklift-ova-provider-server-container" \
+        name="forklift/forklift-ova-provider-server-rhel9" \
+        license="Apache License 2.0" \
+        io.k8s.display-name="Forklift" \
+        io.k8s.description="Forklift - OVA Provider Server" \
+        io.openshift.tags="migration,mtv,forklift" \
+        io.openshift.build.commit.id="${CI_FORKLIFT_UPSTREAM_COMMIT}" \
+        io.openshift.build.source-location="${CI_FORKLIFT_UPSTREAM_URL}" \
+        io.openshift.build.commit.url="https://github.com/kubev2v/forklift/commit/${CI_FORKLIFT_UPSTREAM_COMMIT}" \
+        summary="Forklift - OVA Provider Server" \
+        description="Forklift - OVA Provider Server" \
+        maintainer="Forklift by Konveyor Community <forklift-dev@googlegroups.com>"
```

Comment:
We took the name from the u/s operator 
![image](https://github.com/user-attachments/assets/26d421c6-c26b-4349-bbd5-29a303c1ebd3)

Reply:
fyi @yaacov